#diagnostic
string DeviceName
string NodeName
uint8 System
uint8 SubSystem
uint8 Component
uint8 DiagnosticType
uint8 Level
uint8 DiagnosticMessage
string Description
